sdk/communication/Azure.Communication.CallAutomation/src/CallConnection.cs

```diff
@@ -840,5 +840,99 @@ public async virtual Task<Response<UnmuteParticipantsResult>> UnmuteParticipants
                 throw;
             }
         }
+
+        /// <summary>
+        /// Cancel add participant operation.
+        /// </summary>
+        /// <param name="invitationId"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public virtual Task<Response<CancelAddParticipantResult>> CancelAddParticipantAsync(string invitationId, CancellationToken cancellationToken = default)
+        {
+            return CancelAddParticipantAsync(new CancelAddParticipantOptions(invitationId), cancellationToken);
+        }
+
+        /// <summary>
+        /// Cancel add participant operation.
+        /// </summary>
+        /// <param name="options"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public async virtual Task<Response<CancelAddParticipantResult>> CancelAddParticipantAsync(CancelAddParticipantOptions options, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(CallConnection)}.{nameof(CancelAddParticipant)}");
+            scope.Start();
+
+            try
+            {
+                if (options == null)
+                {
+                    throw new ArgumentNullException(nameof(options));
+                }
+
+                var request = new CancelAddParticipantRequestInternal(options.InvitationId)
+                {
+                    OperationContext = options.OperationContext,
```

Comment:
IN order to work with EventProcessor's intent, this need to be updated to be populated if it is empty. See this line from other method that utilizes eventProcessor:

> request.OperationContext = options.OperationContext == default ? Guid.NewGuid().ToString() : options.OperationContext;

Reply:
Thanks! Fixed.